fiftyone/core/dataset.py

```diff
@@ -105,6 +107,27 @@ def get_default_dataset_name():
     return name
 
 
+def make_unique_dataset_name(root):
```

Comment:
It doesn't look this is used anywhere, only forwarded in `__public__.py`. Intentional?

Reply:
Yeah good catch. At one point it was being used in the implementation here, but then I removed it. Still nice to have around as a utility method though :)